test/js/web/worker-fixture-env.js

```diff
@@ -1,12 +0,0 @@
-import worker_threads from "worker_threads";
-
-if (worker_threads.isMainThread) throw new Error("worker_threads.isMainThread is wrong");
```

Comment:
You mean the test has been removed upstream? 
I've used `git mv` twice on few files and dunno if it's a git bug or expected behavior but the link to the original file and its changes is broken for these files.
I will inspect upstream changes, address review comments then ask for review in a few days.

Reply:
it was deleted here where i commented and not re-added in `test/js/web/workers/worker-fixture-env.js` where it was moved.

Reply:
The check has been moved to `node:worker_threads` tests worker implementation because `isMainThread` is not part of the web worker API

Tests entanglement between worker_threads and web worker is not complete yet.

I will do another PR to move worker_threads tests and workers code to node:worker_threads test directory.

Reply:
i guess this makes sense, but it is something bun allows, but i don't think we explicilty document this behavior anywhere.